### ISSUE TITLE
Replace iconv TRANSLIT

### ISSUE THREAD

**Issue by Kdecherf** [wallabag/wallabag#5377]
We use `iconv()` and `//TRANSLIT` in some occasions to remove things like accent from characters.

Unfortunately starting with gnu-libiconv 1.16, glibc and musl (_e.g. on alpine images_) builds have different ways to handle transliteration.

A better alternative would be to move to something like Symfony's AsciiSlugger, available starting with Symfony 5.

**Comment by tcitworld** [comment]
Also there's this https://stackoverflow.com/a/35178027

**Comment by tcitworld** [comment]
And there's `mb_convert_encoding` for the rest of the uses.

**Comment by Kdecherf** [comment]
@tcitworld As far as I know we don't use `iconv()` outside of the title transliteration to make a filename.

**Comment by tcitworld** [comment]
https://github.com/wallabag/wallabag/blob/5344db79246c00828b82f8befe6884a552a372fa/src/Wallabag/CoreBundle/Helper/EntriesExport.php#L513-L516

**Comment by Kdecherf** [comment]
Yeah, literally "we don't use iconv() outside of the title transliteration to make a filename" 😅

**Comment by tcitworld** [comment]
I read the opposite, sorry. :sweat_smile: 

I meant to say there's two different uses : making sure the title only contains correct UTF8 characters (for display) and converting the title to ASCII (for file export).
